GravatarApp/AppRoot/GravatarAppApp.swift

```diff
@@ -1,11 +1,25 @@
 import OAuth
+import SwiftData
 import SwiftUI
 
 @main
 struct GravatarAppApp: App {
+    var modelContext: ModelContext
+    @State private var unrecoberableError: Error?
+
+    init() {
+        do {
+            self.modelContext = try ModelContext(ModelContainer(for: ProfileStore.self))
+        } catch {
+            print("Error creating model context: \(error)")
+            resetSwiftDataStore()
+            modelContext = try! ModelContext(ModelContainer(for: ProfileStore.self))
+        }
+    }
```

Comment:
I'm not sure what to do with this `try!` here. If this fails, it's really an unrecoverable error.

Deleing the database file with `resetSwiftDataStore()`, has the same effect than re-installing the app, so this last try! should technically never fail.